generator/src/test/java/com/scottlogic/deg/generator/inputs/ProfileReaderTests.java

```diff
@@ -30,7 +29,7 @@
     private String json;
     private Profile profile;
 
-    @Before
+    @BeforeEach
```

Comment:
I presume that `@Before` (junit) and `@BeforeEach' (jupiter) are the same.

Reply:
`@BeforeEach` is equivalent to `@Before` and `@BeforeAll` is equivalent to `@BeforeClass`

Reply:
looks correct:
junit: http://junit.sourceforge.net/javadoc/org/junit/Before.html
jupiter: https://junit.org/junit5/docs/5.0.2/api/org/junit/jupiter/api/BeforeEach.html
comparison: https://sormuras.github.io/blog/2018-09-13-junit-4-core-vs-jupiter-api.html#life-cycle-hooks